pipeline/e2e-tests.sh

```diff
@@ -16,15 +16,15 @@
 
 set +x
 
-id=$(grep id ./env.json | cut -d = -f2)
-# Store the testbed ID in a local file on the workspace.
-echo "$id" > pipeline/placeholder.id
-
 vcIp=$(grep vcIp ./env.json | cut -d = -f2)
 vimUsername=$(grep vimUsername ./env.json | cut -d = -f2)
 vimPassword=$(grep vimPassword ./env.json | cut -d = -f2)
 vcRootPassword=$(grep vcRootPassword ./env.json | cut -d = -f2)
 
+kubeconfigcontent=$(cat ./sv_kubeconfig_content.yaml)
+echo "$kubeconfigcontent" > sv_kubeconfig_content.yaml
+kubeconfig_path="$(pwd)/sv_kubeconfig_content.yaml"
```

Comment:
Can we use a consistent naming format? I see that we have used camel casing for vcIp, vimUsername, etc, but not here. 

@divyenpatel does linter for shell scripts catch such things?

Reply:
Fixed it.